README.md

```diff
@@ -9,8 +9,8 @@ Thanks for everything! See you next game.
 
 ## Requirements
 
-* Node 19+
-* Python 3.9+ (optional - for refreshing the store for IA)
+-   Node 19+
+-   Python 3.9+ (optional - for refreshing the store for IA)
```

Comment:
Is it better to use `-` instead of `*` when making bullet points?

Reply:
This is one of the things Prettier is [opinionated](https://prettier.io/docs/en/option-philosophy.html) about. I think they ran a poll one time and just went with what was most popular.

Reply:
By the way, is Node 19+ actually a requirement? Seems to run okay in 18, and the current LTS is only 18.16.0

Reply:
It's not a requirement, it's just the version that I'm currently running to it was easiest to just default to that 😇